services/priceService.ts

```diff
@@ -66,25 +65,70 @@ export async function upsertCurrentPricesForNetworkId (responseData: IResponseDa
   await upsertPricesForNetworkId(responseData, networkId, 0)
 }
 
-export async function getPriceForDayTicker (day: moment.Moment, ticker: string, attempt: number = 1): Promise<IResponseData | null> {
+function validateTicker (ticker: string): void {
+  if (!Object.values(TICKERS).includes(ticker)) {
+    throw new Error(RESPONSE_MESSAGES.INVALID_TICKER_400.message)
+  }
+}
+
+function validatePriceAPIUrlAndToken (): void {
   if (appInfo.priceAPIURL === '') {
     throw new Error(RESPONSE_MESSAGES.MISSING_PRICE_API_URL_400.message)
   }
-  if (!Object.values(TICKERS).includes(ticker)) {
-    throw new Error(RESPONSE_MESSAGES.INVALID_TICKER_400.message)
+  if (appInfo.priceAPIToken === '') {
+    throw new Error(RESPONSE_MESSAGES.MISSING_PRICE_API_TOKEN_400.message)
   }
+}
+
+function getPriceByDayURL (day: moment.Moment, ticker: string): string {
```

Comment:
This function sounds like it returns a price value. Maybe reword to something like `getURLForPriceByDayRequest` or something like that

Reply:
Done